app/models/organization_resolver.rb

```diff
@@ -48,6 +49,25 @@ def organization_tag_names
     @organization&.tags || []
   end
 
+  def entities
+    [
+      @organization,
+      @email,
+      @location,
+      @organization_name,
+      @phone_number,
+      @website
+    ]
+  end
+
+  def source_names
+    entities.reduce([]) do |memo, entity|
+      next entity if entity.nil?
+      name = entity.versions.last.actor.import.source.name
```

Comment:
It would be nice to move this up to a higher class. Not exactly sure how that would look. Maybe in ApplicationRecord.